pilot/cmd/istioctl/inject.go

```diff
@@ -230,7 +274,8 @@ func init() {
 
 	injectCmd.PersistentFlags().StringVar(&meshConfigFile, "meshConfigFile", "",
 		"mesh configuration filename. Takes precedence over --meshConfigMapName if set")
-	injectCmd.PersistentFlags().StringVar(&injectConfigFile, "injectConfigFile", "", "injection configuration filename")
+	injectCmd.PersistentFlags().StringVar(&injectConfigFile, "injectConfigFile", "",
+		"injection configuration filename. Takes precedence over --injectConfigMapName if set")
```

Comment:
Can you add a new case after line 159 to check that `--injectConfigMapName` and `--injectConfigFile` are not set at the same time?

Reply:
will do